datadog_checks_dev/tests/tooling/manifest_validator/input_constants.py

```diff
@@ -196,11 +198,208 @@
 
 DISPLAY_ON_PUBLIC_VALID_MANIFEST = JSONDict({"display_on_public_website": True})
 
-INVALID_DIFF = 'app_id id'
+IMMUTABLE_ATTRIBUTES_V1_MANIFEST = {"manifest_version": "1.0.0"}
+
+IMMUTABLE_ATTRIBUTES_V2_MANIFEST = JSONDict({"manifest_version": "2.0.0"})
+
+IMMUTABLE_ATTRIBUTES_PREV_MANIFEST_INVALID = json.dumps(
```

Comment:
Whats the invalid part of this manifest?

Reply:
I changed the name of some of the input constants to more clearly describe their use.